pyzbar/pyzbar.py

```diff
@@ -11,14 +11,15 @@
     zbar_image_create, zbar_image_destroy, zbar_image_set_format,
     zbar_image_set_size, zbar_image_set_data, zbar_scan_image,
     zbar_image_first_symbol, zbar_symbol_get_data,
+    zbar_symbol_get_loc_size, zbar_symbol_get_loc_x, zbar_symbol_get_loc_y,
     zbar_symbol_next, ZBarConfig, ZBarSymbol, EXTERNAL_DEPENDENCIES
 )
 
 __all__ = ['decode', 'EXTERNAL_DEPENDENCIES']
 
 
```

Comment:
Suggest that we return a single rectangle that describes the bounding box around the detected barcode:

```
# A rectangle
Rect = namedtuple('Rect', ['left', 'top', 'width', 'height'])

 # Results of reading a barcode
Decoded = namedtuple('Decoded', ['data', 'type', 'rect'])
```